sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -745,18 +746,29 @@ def process_bundle_progress(
           instruction_id=instruction_id, error=traceback.format_exc())
     if processor:
       monitoring_infos = processor.monitoring_infos()
+      consuming_received_data = \
+        processor.consuming_received_data
+      return beam_fn_api_pb2.InstructionResponse(
```

Comment:
Don't duplicate the whole return; just set `consuming_received_data = None` (or `False`) just as was done with monitoring infos.

Reply:
Done. Thanks!

It wouldn't let me add None. So I added false and edited unit tests accordingly.